docs/source/faq.rst

```diff
@@ -129,9 +129,13 @@ One of the first settings that usually needs to be bumped is the maximum number
 of open file descriptors for a given process. For the confused out there,
 remember that Unices treat sockets as files.
 
-::
+.. warning:: ``sudo ulimit`` may not work
```

Comment:
the `.. warning:: ` must be related to the paragraph below. thre is no real need for this line as like you said it will not work anyway.